src/lib.rs

```diff
@@ -1,9 +1,15 @@
+/* These are the public namespaces shown in import paths e.g.
+    use momento::config::configurations::laptop;
+    use momento::cache::{Get, GetRequest, Set, SetRequest};
+*/
 pub mod auth;
+pub mod cache;
 /// Configuration structs for the Momento clients.
 pub mod config;
+pub mod errors;
 pub mod preview;
-pub mod requests;
 pub mod response;
+/*************************************************************/
```

Comment:
are we okay with these namespaces?

hadn't rearranged anything related to topics yet, but should we rename it from "preview" to just "topics"?

Reply:
good callout, ty

let's make another pass to deal with topics after we are happy with cache stuff.

response namespace will go away once we remove the old client, i think.

we should check out auth and config and see if we want to re-export stuff from them either directly under `momento`, or into `cache` (e.g. for configuration, which is probably going to be cache-specific). Don't feel obligated to hit those in this PR, but maybe make a ticket for it.

Reply:
captured in ticket https://github.com/momentohq/client-sdk-rust/issues/222